mindjunkies/forums/views.py

```
@@ -1,129 +1,176 @@
+
+
+class ForumHomeView(LoginRequiredMixin, CourseContextMixin, TemplateView):
     template_name = "forums/forum_home.html"
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        # Get course_slug from URL
-        course_slug = self.kwargs.get("course_slug", "")
-        context["course_slug"] = course_slug
+        course = context["course"]
```

Ruff (0.8.2)

Local variable `course` is assigned to but never used

Remove assignment to unused variable `course`

(F841)